### DISCUSSION TITLE
Suggestions missing?

### DISCUSSION THREAD

**Discussion by atom-smasher** [AnySoftKeyboard/AnySoftKeyboard#3108] [Q&A]
I'm playing around and seeing if AnySoftKeyboard is what I'm looking for.

In a note/text editor, or a messaging app, I'm seeing the "Suggestions" bar is appearing as desired. Good so far.

If I open a browser and type into a "form" field, again I get the desired behaviour with a "Suggestions" bar.

But in the same browser, if I type into the address-bar or a "text" field, there's no "Suggestions" bar. This is not what I want. I want a "Suggestions" bar wherever I'm typing.

Suggestions?

**Comment by menny** [comment]
I think that this is because AnySoftKeyboard assumes this is a URL input field which does not require suggestions. Can you yell me which browser that is?
